CHANGELOG.md

```diff
@@ -6,6 +6,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- New metrics:
+  - `tnt_vinyl_spaces` (same as `tnt_space_count`)
+  - `tnt_net_sent_bytes`  (same as `tnt_net_sent_total`)
+  - `tnt_net_received_bytes` (same as `tnt_net_received_total`)
+  - `tnt_net_connections_amount` (same as `tnt_net_connections_total`)
+  - `tnt_net_requests_amount` (same as `tnt_net_requests_total`)
+  - `tnt_fiber_amount` (same as `tnt_fiber_count`)
+  - `tnt_stats_op_amount` (same as `tnt_stats_op_total`)
+  - `lj_gc_memory` (same as `lj_gc_total`)
+  - `tnt_cpu_number` (same as `tnt_cpu_count`)
```

Comment:
It is not obvious what is meant with "cpu_number" until you read docs. But I don't have a better proposal for now